CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Security
 - Upgrade github-pages in docs/Gemfile to resolve CVE-2021-28834 in kramdown dependency [cyberark/conjur#2099](https://github.com/cyberark/conjur/issues/2099)
 
+### Added
+- File `API_VERSION` containing the current Conjur API Version, which corresponds to a
+  release of the [Conjur OpenAPI Spec](https://github.com/cyberark/conjur-openapi-spec).
```

Comment:
this also needs the link to the PR: [PR cyberark/conjur#2107](https://github.com/cyberark/conjur/pull/2107)